spaceship/lib/spaceship/test_flight/build_trains.rb

```diff
@@ -13,12 +13,76 @@ class BuildTrains < Base
     # See `Spaceship::TestFlight::Build#reload`
 
     def self.all(app_id: nil, platform: nil, retry_count: 3)
-      data = client.get_build_trains(app_id: app_id, platform: platform)
-      trains = {}
+      client = Spaceship::ConnectAPI::Base.client
```

Comment:
If this is temporary, is our plan to eventually remove the entire model or just this new code-path? If we would like to remove a code-path, can all of this be extracted into its own method so that later it can be removed easier?

Reply:
I think we are going to remove these models if possible 😬